assets/js/googlesitekit/datastore/user/authentication.test.js

```diff
@@ -0,0 +1,358 @@
+/**
+ * core/user Data store: Authentication info tests.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+
+/**
+ * Internal dependencies
+ */
+import API from 'googlesitekit-api';
+import {
+	createTestRegistry,
+	muteConsole,
+	subscribeUntil,
+	unsubscribeFromAll,
+} from '../../../../../tests/js/utils';
+import { STORE_NAME } from './constants';
+
+describe( 'core/user authentication', () => {
+	const coreUserDataExpectedResponse = { authenticated: true, requiredScopes: [], grantedScopes: [] };
+	const coreUserDataEndpointRegExp = /^\/google-site-kit\/v1\/core\/user\/data\/authentication/;
+	let apiFetchSpy;
+	let registry;
+	let store;
+
+	beforeAll( () => {
+		API.setUsingCache( false );
+	} );
+
+	beforeEach( () => {
+		registry = createTestRegistry();
+		store = registry.stores[ STORE_NAME ].store;
+
+		apiFetchSpy = jest.spyOn( { apiFetch }, 'apiFetch' );
+	} );
+
+	afterAll( () => {
+		API.setUsingCache( true );
+	} );
+
+	afterEach( () => {
+		unsubscribeFromAll( registry );
+		apiFetchSpy.mockRestore();
+	} );
+
+	describe( 'actions', () => {
+		describe( 'fetchAuthentication', () => {
+			it( 'does not require any params', () => {
+				expect( () => {
+					registry.dispatch( STORE_NAME ).fetchAuthentication();
+				} ).not.toThrow();
+			} );
+		} );
```

Comment:
We can probably remove this test. We don't have tests for most fetch actions I think because they're implicitly covered when testing "uses a resolver to make a network request".

I'm not against keeping it, but as it is it will log an error in the console because the request will fail as there's no fetch mock in place.